src/test/java/uk/gov/justice/digital/zone/Fixtures.java

```diff
@@ -19,35 +21,88 @@ public class Fixtures {
     public static final String TABLE_NAME = "agency_internal_locations";
     public static final String TABLE_OPERATION = "load";
     public static final String ROW_CONVERTER = "row_converter";
-    public static final String RAW_DATA = "{}";
 
-    public static final StructType ROW_SCHEMA = new StructType()
-            .add(SOURCE, StringType, false)
-            .add(TABLE, StringType, false)
-            .add(OPERATION, StringType, false);
+    public static final String RECORD_KEY_1 = "record-1";
+    public static final String RECORD_KEY_2 = "record-2";
+    public static final String RECORD_KEY_3 = "record-3";
+    public static final String RECORD_KEY_4 = "record-4";
+    public static final String RECORD_KEY_5 = "record-5";
+    public static final String RECORD_KEY_6 = "record-6";
+    public static final String RECORD_KEY_7 = "record-7";
 
-    public static final GenericRowWithSchema dataMigrationEventRow = new GenericRowWithSchema(
-            new Object[] { TABLE_SOURCE, TABLE_NAME, TABLE_OPERATION },
-            ROW_SCHEMA
-        );
+    public static final String PRIMARY_KEY_PLACEHOLDER = "<PRIMARY-KEY>";
 
-    public static final GenericRowWithSchema dataMigrationEventRowWithInvalidOperation = new GenericRowWithSchema(
-            new Object[] { TABLE_SOURCE, TABLE_NAME, "makeTea" },
-            ROW_SCHEMA
-    );
+    public static String PRIMARY_KEY_FIELD = "primary-key";
 
+    public static String STRING_FIELD_KEY = "string-key";
```

Comment:
can we move all these final values to abstract class and no need of private constructor Fixtures()?

Reply:
Done 👍